### DISCUSSION TITLE
Change author metadata

### DISCUSSION THREAD

**Discussion by smorele** [getpelican/pelican#3168] [Q&A]
Hello,
I need to add metadata (i.e: :avatar:) to author but I do not find how to. I've tested several things such as creating a dedicated profile page into /content/author/my-author.rst but content is not used my custom html.

Is there a way to do it ?

**Comment by jorgesumle** [comment]
Puedes crear un diccionario como el siguiente en `pelicanconf.py`:

```
AUTORES = {'Jorge': 'path-to-avatar.png',
           'Daniel': 'path2avatar.png'}
```

En la plantilla del autor `templates/author.html` podrías poner algo así:

```
{% if author in AUTORES: %}
    <img src="{{ AUTORES[author] }}" alt="avatar de {{author}}">
{% endif %}
```
